pkg/cmd/attestation/trustedroot/trustedroot.go

```diff
@@ -74,6 +80,7 @@ func NewTrustedRootCmd(f *cmdutil.Factory, runF func(*Options) error) *cobra.Com
 	trustedRootCmd.Flags().StringVarP(&opts.TufRootPath, "tuf-root", "", "", "Path to the TUF root.json file on disk")
 	trustedRootCmd.MarkFlagsRequiredTogether("tuf-url", "tuf-root")
 	trustedRootCmd.Flags().BoolVarP(&opts.VerifyOnly, "verify-only", "", false, "Don't output trusted_root.jsonl contents")
+	trustedRootCmd.Flags().StringVarP(&opts.Hostname, "hostname", "", "", "Configure host to use")
```

Comment:
You are correct. This PR only adds support for the hostname flag, the real work will come in a separate PR. 

The same is for the `verify` command, I have this quite big change https://github.com/cli/cli/compare/trunk...kommendorkapten:policy-tenancy?expand=1 that modifies the policy to work with the updated hostname. 

I opted to have a few but smaller PR as it would be easier to review.

Reply:
Note that the above PR is based on this branch, so the change set looks a bit bigger, only the last commit is the verification/policy change.

Reply:
Ah ok. What is your confidence that that PR will be in by our usual tuesday release schedule? If it's not extremely high, maybe we could mark the flags as hidden so they don't confuse users by being non-functional?

Reply:
Ah, yes that is a great idea. Let me figure out how to hide the flag.

I have two more PRs I need to finish (verify is ready for review, but need to fix trusted root and inspect command too).

Reply:
I think I'll have the code ready until Tuesday, but not suer we will get it all reviewed in time though.

Reply:
Something like:

```go
	_ = cmd.Flags().MarkHidden("hostname")
```

Best to do this I think.